tests/model_hub_tests/pytorch/hf_transformers_models

```diff
@@ -257,7 +257,7 @@ microsoft/xclip-base-patch32,xclip
 microsoft/xprophetnet-large-wiki100-cased,xlm-prophetnet
 miguelvictor/python-fromzero-lstmlm,lstmlm,skip,Load problem
 mingzi151/test-hf-wav2vec2bert,wav2vec2bert,skip,Load problem
-mistralai/Mistral-7B-v0.1,mistral
+mistralai/Mistral-7B-v0.1,mistral,skip,Model is private
```

Comment:
can we use tiny test model instead? 
https://huggingface.co/HuggingFaceM4/tiny-random-MistralForCausalLM

or use one of fine-tuned version (zephyr, neural-chat, notus)